core/assets/bundles/bundle_es.properties

```diff
@@ -779,9 +779,9 @@ sector.atlas.description = Este sector contiene terreno variado, por lo que nece
 sector.split.description = La baja actividad enemiga de este sector lo hace ideal para experimentar con nuevas tecnologías de transporte.
 sector.basin.description = {Temporal}\n\nEl último sector por ahora. Considéralo un nivel de desafío - se añadirán más sectores en futuras versiones.
 sector.ravine.description = No se han detectado núcleos enemigos en este sector, a pesar de ser una de sus principales ruta de transporte. Prepárate para confrontar una gran variedad de fuerzas hostiles.\nProduce [accent]aleación eléctrica[]. Construye torretas "[accent]Afflict[]".
-sector.caldera-erekir.description = Esto es una descripción :)
-sector.stronghold.description = WiP Description.
```

Comment:
I think it would be better to not translate these descriptions.

Reply:
ok, I'll leave them as they were before.